src/commcare_cloud/commands/migrations/copy_files.py

```diff
@@ -218,12 +220,10 @@ def prepare_file_copy_scripts(target_host, source_file_configs, script_root):
 
 def copy_scripts_to_target_host(target_host, script_root, environment, ansible_context):
     local_files_path = os.path.join(script_root, target_host)
-
     destination_path = os.path.join('/tmp', REMOTE_MIGRATION_ROOT)
```

Comment:
I don't love that `destination_path` is defined both here and within `remove_scripts` as it seems prone to error. Maybe adding it as an optional parameter to `remove_scripts` could work?

Reply:
27d6a00732dd3d9013db72eb2fa3721651959b16

Reply:
Even better!